crates/ra_assists/src/ast_editor.rs

```diff
@@ -19,7 +19,7 @@ pub struct AstEditor<N: AstNode> {
 }
 
 impl<N: AstNode> AstEditor<N> {
-    pub fn new(node: N) -> AstEditor<N>
+    pub fn new(node: N) -> Self
```

Comment:
I think `Self` should be avoided most of the time: why make the human do extra work?